### DISCUSSION TITLE
Is it possible to set this line in pyproject.toml to use a default during init?

### DISCUSSION THREAD

**Discussion by ughstudios** [python-poetry/poetry#7571] [Q&A]
```
[tool.poetry.dependencies]
python = "^3.10"
```

It keeps being set to something I don't want. Is it possible to have it  load from a file, or use an environment variable, or set some sort of default?

Here is what I want:

```poetry init```

Should create this file, every time:

[tool.poetry.dependencies]
python = "my_version_string_loaded_from_a_file()"

Is this possible?

**Comment by PabloEmidio** [comment]
You can use the `--python` argument with the command to choose the python version. [See docs](https://python-poetry.org/docs/cli/#init)

**Reply by ughstudios** [reply]
What can I do to have it use a wild card for the python version?

**Reply by PabloEmidio** [reply]
You'll have to use the argument as a literal string, e.g `poetry init --python="3.11.*"`
